src/dict.c

```diff
@@ -1308,6 +1414,45 @@ int dictTest(int argc, char **argv, int accurate) {
     }
     end_benchmark("Accessing missing");
 
+    dictX *dictX = dictEvolution(dict);
```

Comment:
Would be practical and clearer perhaps to implement `dictXAdd()` and do a loop of `dictUnlink()` and `dictXAdd()` instead of the more complex `dictEvolution()` function.

Reply:
Thanks for the suggestion, I will do that, actually dict and dictX are one and the same thing, in this test I made them parallel to give you guys a lot of confusion, because I don't want to pollute the rest of dict.c for now, so I made them parallel. Here dictEvolution, dictRewind are temporary, I will improve them according to your suggestion.